Assets/js/project-leads.js

```diff
@@ -0,0 +1,28 @@
+const projectLeads = ()=>{
+    document.addEventListener('DOMContentLoaded', function() {
+        const showMoreButton = document.querySelector('.show-more-button');
+        const experts = document.querySelectorAll('.expert');
+        let visibleExpertsCount = 5;
+
+        // Initially hiding extra experts
+        experts.forEach((expert, index) => {
```

Comment:
@dkdeepesh Add a null check on it because we are not going to use this component on other pages. So, on other pages it'll throw an error.
Cc:- @jvaishnavrp  @mathrik